app/controllers/events/view/tickets/orders/list.js

```diff
@@ -74,6 +74,26 @@ export default Controller.extend({
         .finally(() => {
           this.set('isLoading', false);
         });
+    },
+    async resendConfirmation(order) {
+      let payload = {};
+      try {
+        payload = {
+          'data': {
+            'order' : order.identifier,
+            'user'  : this.authManager.currentUser.email
+          }
+        };
+        await this.loader.post('orders/resend-email', payload);
+        this.notify.success(this.l10n.t('Email confirmation has been sent to attendees successfully'));
+      } catch (error) {
+        if (error.status === 429) {
+          this.notify.error(this.l10n.t('Only 5 resend actions are allowed in a minute'));
```

Comment:
What if the error is something other than the two you specified.
Comparing error returned by server with the codes is not a optimisable solution its more like a workaround. @uds5501 is this the only working way for this problem ?

Reply:
@kushthedude There's no context of optimality here. @uds5501 Just add an else statement for other errors & this is good to go :D